docs/source/cli.rst

```diff
@@ -31,7 +31,7 @@ Read it
     optional arguments:
       -h, --help            show this help message and exit
       --pattern PATH, -p PATH
-                            paths to pattern definition files
+                            paths to pattern definition files or modules
```

Comment:
I guess I can only come up with:

1. I don't have to change the examples?
1. Supporting paths gives you tab completion, where module names really don't